integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KeytoolCertificateGenerator.java

```diff
@@ -21,7 +21,7 @@
 
 public class KeytoolCertificateGenerator {
     private static String password;
-    private static final String certsDirectory = "certs";
+    private static final String certsDirectory = "/tmp/kproxy";
```

Comment:
I think we should be injecting the `certsDirectory` and using [`@TempDir`](https://junit.org/junit5/docs/5.4.1/api/org/junit/jupiter/api/io/TempDir.html) in tests to generate it.